app/icahost.go

```diff
@@ -0,0 +1,17 @@
+package app
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	icahosttypes "github.com/cosmos/ibc-go/v8/modules/apps/27-interchain-accounts/host/types"
+	icatypes "github.com/cosmos/ibc-go/v8/modules/apps/27-interchain-accounts/types"
+)
+
+type ICAHostMockSubspace struct {
```

Comment:
```suggestion
// ICAHostMockSubspace is a mock implementation to workaround the migration process, because we have nothing to migrate from.
type ICAHostMockSubspace struct {
```